build_all.sh

```diff
@@ -42,7 +48,20 @@ if [[ "$JETPACK_VERSION" == "6.0" ]]; then
     cd $SRCS
     export KERNEL_HEADERS=$SRCS/kernel/kernel-jammy-src
     ln -sf $TEGRA_KERNEL_OUT $SRCS/out
-    make ARCH=arm64 -C kernel
+    if [[ "$DEVDBG" == "1" ]]; then
+        cd $KERNEL_HEADERS
+        make ARCH=arm64 defconfig
```

Comment:
Can you explain a bit about all the added lines?
Thanks :)

Reply:
Maybe add some comments?

Reply:
Added comments.